stats/stats/src/charts/db_interaction/write.rs

```diff
@@ -56,6 +56,14 @@ where
     Ok(())
 }
 
+pub async fn clear_all_chart_data<C: ConnectionTrait>(db: &C, chart_id: i32) -> Result<(), DbErr> {
+    chart_data::Entity::delete_many()
+        .filter(chart_data::Column::ChartId.eq(chart_id))
+        .exec(db)
+        .await?;
+    Ok(())
+}
```

Comment:
nah its ok

Reply:
@coderabbitai please learn that it's impossible to begin transaction if function take argument as `ConnectionTrait` since it can be DatabaseConnection or TransactionConnection. also if transaction performs only one sql statement it's useless to begin transaction.

eat more carrots to be smarter boy